LAMMPS/KMC_Thiophene/LAMMPS_KMC.ipynb

```diff
@@ -0,0 +1,492 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The option 'map_file' on file 'OPTIONFILES/mapchecker.xml' has been set to 'system.xml'\n"
+     ]
+    }
+   ],
+   "source": [
+    "from xml_editor import edit_calculator\n",
+    "edit_calculator(\"mapchecker\", \"map_file\", \"system.xml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can run the checker as follows,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "==================================================\n",
+      "========   VOTCA (http://www.votca.org)   ========\n",
+      "==================================================\n",
+      "\n",
+      "please submit bugs to bugs@votca.org\n",
+      "\n",
+      "xtp_run, version 1.7-dev gitid: 6e47b87 (dirty) (compiled Jun  2 2020, 10:18:25)\n",
+      "votca_csg, version 1.7-dev gitid: 6e47b87 (dirty) (compiled Jun  2 2020, 10:14:08)\n",
+      "votca_tools, version 1.7-dev gitid: 6e47b87 (dirty) (compiled Jun  2 2020, 10:13:13)\n",
+      "\n",
+      "Initializing calculator\n",
+      "... mapchecker \n",
+      "1 frames in statefile, Ids are: 10000 \n",
+      "Starting at frame 10000\n",
+      "Evaluating frame 10000\n",
+      "Import MD Topology (i.e. frame 10000) from state.hdf5\n",
+      ".... \n",
+      "... mapchecker \n",
+      "Writing segments to md_segments_step_10000.pdb\n",
+      "Writing qmmolecules to qm_segments_n_step_10000.pdb\n",
+      "Writing polarsegments to mp_segments_e_step_10000.pdb\n",
+      "Writing polarsegments to mp_segments_h_step_10000.pdb\n",
+      "\n",
+      "Changes have not been written to state file.\n"
+     ]
+    }
+   ],
+   "source": [
+    "!xtp_run -e mapchecker -o OPTIONFILES/mapchecker.xml -f state.hdf5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Site energies and pair energy differences\n",
+    "We will compute the histrogram using `resolution_sites` of 0.03 eV. See [eanalize options and defaults](https://votca.github.io/eanalyze.html) for more information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The option 'resolution_sites' on file 'OPTIONFILES/eanalyze.xml' has been set to '0.03'\n",
+      "==================================================\n",
+      "========   VOTCA (http://www.votca.org)   ========\n",
+      "==================================================\n",
+      "\n",
+      "please submit bugs to bugs@votca.org\n",
+      "\n",
+      "xtp_run, version 1.7-dev gitid: 6e47b87 (dirty) (compiled Jun  2 2020, 10:18:25)\n",
```

Comment:
Output is still there.

Reply:
Shall I remove all the output from all the jupyter cell? I was thing that we only one to remove long outputs and leave the short ones

Reply:
@JensWehner what do you think, I just don't like the idea of having hard-coded git hashes in here, which change all the time.

Reply:
I agree with Christoph, nothing should be in the repo which can be created by just running it unless it really takes a long time. Can we not create the documentation from the notebook by executing it?

Reply:
OK, I am cleaning the cell of the jupyter notebooks and we run them in the CI